docs/tutorials/11_working_with_classical_optimization_solvers.ipynb

```diff
@@ -0,0 +1,469 @@
+{
+ "metadata": {
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.8.11"
+  },
+  "orig_nbformat": 4,
+  "kernelspec": {
+   "name": "python3",
+   "display_name": "Python 3.8.11 64-bit ('dev38': venv)"
+  },
+  "interpreter": {
+   "hash": "55198b233f406dc59b712acd847cdd7a571fa057c585d106ba05aad0d31ffb93"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 2,
+ "cells": [
+  {
+   "source": [
+    "# Working with Classical Optimization Solvers"
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "source": [
+    "We can use classical optimization solvers (CPLEX and Gurobi) with Qiskit Optimization.\n",
+    "Docplex and Gurobipy are the Python APIs for CPLEX and Gurobi, respectively.\n",
+    "We can load and save an optimization model by Docplex and Gurobipy and can apply CPLEX and Gurobi to `QuadraticProgram`.\n",
+    "\n",
+    "If you want to use the solver of CPLEX, you need to install `pip install 'qiskit-optimization[cplex]'`. Docplex is supposed to be installed with Qiskit Optimization.\n",
```

Comment:
Would this `If you want to use the solver of CPLEX...` be better as `If you want to use the CPLEX solver`

Reply:
Thanks. I updated it.